src/hf_hydrodata/gridded.py

```diff
@@ -1251,8 +1251,8 @@ def _apply_mask(data, entry, options):
     The second option masks only against ocean and outer HUC bounds. The first option masks with internal HUC boundaries.
     """
 
-    if options.get("dataset") == "huc_mapping":
-        # Do not mask the mask
+    if options.get("dataset") == "huc_mapping" or options.get("variable") == "clm_run":
```

Comment:
I changed the code so there are two if statement for the different cases of not masking and better comments for the cases. I used the clm_run variable because there are actually multiple DC entries for clm_run with different file types, but all should be not masked. But your comment is good that this was not clear with the comments.